client/components/password-change/password-change-page.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+import PasswordChange from "./password-change";
+
+const PasswordChangePage = () => {
+  return (
+    <div
+      style={{
```

Comment:
remove this inline style. Since we are using css files to style the components, keep it uniform and move these to css file.